appnav/src/main/kotlin/io/element/android/appnav/loggedin/LoggedInPresenter.kt

```diff
@@ -133,59 +143,6 @@ class LoggedInPresenter(
         currentSlidingSyncVersion == SlidingSyncVersion.Proxy
     }
 
-    private suspend fun ensurePusherIsRegistered(pusherRegistrationState: MutableState<AsyncData<Unit>>) {
```

Comment:
Code moved to `DefaultPushService` so that can be reused by the notification settings screen